#!/bin/sh -e
##  by Joost Yervante Damad <joost@lumatec.be>
##
## DP: do not install Contrib/ including C-BIOS

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -ur openmsx-0.5.0.orig/build/main.mk openmsx-0.5.0/build/main.mk
--- openmsx-0.5.0.orig/build/main.mk	2005-01-06 19:53:35.000000000 +0000
+++ openmsx-0.5.0/build/main.mk	2005-01-06 20:06:24.000000000 +0000
@@ -490,11 +490,6 @@
 	@echo "  Data files..."
 	@mkdir -p $(INSTALL_SHARE_DIR)
 	@cp -rf share/* $(INSTALL_SHARE_DIR)/
-	@echo "  C-BIOS..."
-	@mkdir -p $(INSTALL_SHARE_DIR)/Contrib/cbios
-	@cp -f Contrib/README.cbios $(INSTALL_SHARE_DIR)/Contrib
-	@cp -f $(addprefix Contrib/cbios/,*.BIN *.txt *.rom) \
-		$(INSTALL_SHARE_DIR)/Contrib/cbios
 	@echo "  Documentation..."
 	@mkdir -p $(INSTALL_DOC_DIR)
 	@cp -f README GPL AUTHORS $(INSTALL_DOC_DIR)

